chart/values.schema.json

```
@@ -463,6 +463,21 @@
                             "type": "boolean",
                             "default": false
                         },
+                        "cacheSize": {
+                            "description": "Add cache size for your statsD deployment.",
+                            "type": "integer",
+                            "default": true
+                        },
+                        "cacheType": {
+                            "description": "Add cache type for your statsD deployment (lru/random).",
+                            "type": "string",
+                            "default": true
```

Same here, according to change in values.yaml should be:
```suggestion
                            "default": "lru"
```

---

made changes, thanks!